### PR TITLE
rebrand package publish domain

### PR DESCRIPTION
**Pull request by Sertonix**
This is a small rebrand in the install tab in the settings-view.
It is only a static rebrand for now. I hope that as soon as pulsar is more stable we can make a really generic branding system.

### CODE REVIEW

**Comment by Daeraxa** [comment]
Well the branding API exists so it should be pretty easy to expand some of these direct links to it now as was done with the about package.

File: packages/settings-view/lib/install-panel.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ export default class InstallPanel {
             <div className='text native-key-bindings' tabIndex='-1'>
               <span className='icon icon-question' />
               <span ref='publishedToText'>Packages are published to </span>
-              <a className='link' onclick={this.didClickOpenAtomIo.bind(this)}>atom.io</a>
+              <a className='link' onclick={this.didClickOpenAtomIo.bind(this)}>pulsar-edit.dev</a>
```
Comment (confused-Techie) [inline packages/settings-view/lib/install-panel.js:89]:
```suggestion
              <a className='link' onclick={this.didClickOpenAtomIo.bind(this)}>web.pulsar-edit.dev</a>
```

Like mentioned by @Daeraxa should get this updated to the site where we can actually view packages

**Review by confused-Techie — APPROVED** [review]
LGTM!

**Review by Spiker985 — APPROVED** [review]
Non-code change. Looks good to me

**Pull request merged**